docroot/themes/humsci/humsci_basic/src/scss/elements/_tables.scss

```diff
@@ -0,0 +1,49 @@
+.hb-table-wrap {
+  margin: auto;
+  overflow-x: auto;
+}
+
+table {
+  $border-color: lighten($su-color-driftwood, 15%);
+  // DO NOT set display: block; on a table
+  // Tables need display: table; in order to semantically be a table
+  // display: block; will negatively affect screen reader a11y
+  width: 100%;
+  border: 1px solid $border-color;
+  border-collapse: collapse;
+
+  font-size: 0.8rem;
+  text-align: left;
+
+  @include grid-media('sm') {
+    font-size: 1rem;
+  }
+
+  th, td {
+    border: 1px solid $border-color;
+    padding: 1rem;
+    vertical-align: top;
+  }
+
+  thead th {
+    position: sticky;
+    top: 0;
+    border-color: #fff;
+  }
+
+  thead tr {
+    background: #555;
```

Comment:
If this is the fallback color, I'd make it a variable and put it in the general.variables.scss file

Reply:
you could probably also use: darken($su-color-driftwood, 45%);

Reply:
I ended up finding a good option from the Decanter color variables for this: `$su-color-warm-grey`